benchmarks/benchmark_throughput.py

```diff
@@ -102,9 +135,14 @@ def sample_requests(tokenizer: PreTrainedTokenizerBase,
                 continue
             prompt = _get_prompt_for_image_model(question=prompt, model=model)
 
+        lora_request: Optional[LoRARequest] = None
+        request_tokenizer = tokenizer
+        if args.enable_lora:
+            lora_request, request_tokenizer = get_random_lora_request(args)
+
```

Comment:
There are too many LoRA weights that don't include tokenizer files, which leads to `request_tokenizer` being None. We need to consider this

Reply:
Nice catch ! Added a fallback to use the base tokenizer.